src/screens/Ticketing/Ticket/Carnet/CarnetDetails.tsx

```diff
@@ -81,7 +81,7 @@ export function CarnetDetails(props: {
         )}
         <TicketInfo
           travelRights={travelRights}
-          status={fareContractValidityStatus}
+          status={usedAccessValidityStatus}
```

Comment:
fareContractValidityStatus tells the overall status of the carnet ticket, while usedAccessValidityStatus is if the ticket is currently in use or not. Its value is 'valid' and 'inactive'. Hence passing that down to show zone/non-inspectable symbol and warning.